radicle-node/src/service.rs

```diff
@@ -554,6 +567,38 @@ where
         }
     }
 
+    pub fn ping(&mut self, peer_ip: &net::IpAddr) -> Result<(), peer::SessionError> {
+        let peer = if let Some(peer) = self.sessions.get_mut(peer_ip) {
+            peer
+        } else {
+            return Err(SessionError::NotFound(*peer_ip));
+        };
+
+        let ping_upper = 16;
+        let ponglen = self.rng.u16(1..ping_upper);
```

Comment:
For the upper, we should move it to a constant `MAX_PING_BYTES`, and we should set it to `u16::MAX - 4`, which is the maximum message size minus the message type length, minus the ping size.

Reply:
This got lost.